README.md

```diff
@@ -8,60 +8,122 @@
 
 [![NPM](https://nodei.co/npm/mobx-react-lite.png)](https://www.npmjs.com/package/mobx-react-lite)
 
-**You need React version 16.8.0 and above**
+This is a lighter version of [mobx-react](https://github.com/mobxjs/mobx-react) which supports React **functional components only** and as such makes the library slightly faster and smaller (_only 1.5kB gzipped_). Note however that it is possible to use `<Observer>` inside the render of class components.
+Unlike `mobx-react`, it doesn't `Provider`/`inject`, as `useContext` can be used instead.
+It also doesn't offer
 
-This is a lighter version of [mobx-react](https://github.com/mobxjs/mobx-react) which supports React **functional components only** and as such makes the library slightly faster and smaller (_only 1.5kB gzipped_). In fact `mobx-react@6` has this library as a dependency and builds on top of it.
+## Compatibility table (major versions)
 
-The library does not include any Provider/inject utilities as they can be fully replaced with [React Context](https://mobx-react.js.org/recipes-context). Check out [the migration guide](https://mobx-react.js.org/recipes-migration).
+| mobx | mobx-react-lite | mobx-react<br/>(for class components) | Browser                                        |
```

Comment:
Hm, this is cool, but isn't it too confusing? I mean why to bother with `mobx-react` in here? If people want to use `mobx-react-lite` only they only need to care about MobX version, right? Some might get an impression from this table they also need to install `mobx-react` of a specific version 🤷‍♂️

Reply:
yeah your right, this should probably be somewhere else :)